test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Yeah, and with that opt I see it converted to an `i64.store8`.

But anyhow, we can merge this and leave the mystery alone. But you did actually see the fix in this PR fix something for the wasm backend?